src/olympia/amo/management/commands/generate_jsi18n_files.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.http import HttpRequest
+from django.utils import translation
+from django.views.i18n import javascript_catalog
+
+
+class Command(BaseCommand):
+    help = 'Generate static jsi18n files for each locale we support'
+
+    def handle(self, *args, **options):
+        fake_request = HttpRequest()
+        for lang in settings.AMO_LANGUAGES:
+            filename = os.path.realpath(os.path.join(
```

Comment:
you need `.realpath` here?

Reply:
I'll remove it. My original intention was to add a check to see where the path would end up, in case a rogue language string was added to `AMO_LANGUAGES`, but I didn't follow up with that.

Reply:
f59a2a1015cca3bae6f601b4775ba5c486ddaade